static/app/components/tooltip.tsx

```diff
@@ -234,13 +242,25 @@ export function DO_NOT_USE_TOOLTIP({
       (skipWrapper || typeof triggerChildren.type === 'string')
     ) {
       // Basic DOM nodes can be cloned and have more props applied.
-      return cloneElement(triggerChildren, {...containerProps, ref: setRef});
+      return cloneElement(triggerChildren, {
+        ...containerProps,
+        style: {
+          ...triggerChildren.props.style,
```

Comment:
Does this actually work?

Reply:
I didn't realize you could access the props of children like this 🤔

Reply:
In my testing (just passing a random style object to a tooltip child), it works. Not sure if it's discouraged or not…

Reply:
I think this would be okay, since I'm not changing any of the child's styles, just adding more styles for the underlines and carrying the original ones over.

Reply:
Yeah it's fine